src/styles/utils/_mixin.scss

```diff
@@ -0,0 +1,26 @@
+@mixin square ($size) {
+  width: $size;
+  height: $size;
+  border-radius: 100%;
+}
+
+@mixin animation-second() {
+  animation-name: move;
+  animation-duration: 1s;
+  animation-timing-function: linear;
+  animation-iteration-count: infinite;
+}
+
+@mixin animation-minute() {
+  animation-name: move;
+  animation-duration: 60s;
+  animation-timing-function: linear;
+  animation-iteration-count: infinite;
```

Comment:
type it one line

Reply:
change duration on variable and give it through include, so you will be needed only one mixin

Reply:
@mixin animation($time) {
  animation: move $time linear infinite;
  }